### ISSUE TITLE
'autoencoder.h5'

### ISSUE THREAD

**Issue by rchaaben** [tcloaa/Deep-Portfolio-Theory#3]
Hi,
may I please know which encoder are you using because it seems that you are using a predefined encoder  'autoencoder.h5' and there is no information about it (I'm not able to run the code).
Best regards.

**Comment by tcloaa** [comment]
My bad..

'autoencoder.h5' is a trained model that I used when I was doing this research. I didn't include it in this repo for now... (sorry I was just busy that I don't have time to take care of it right now).

You can build one yourself starting from [here](https://blog.keras.io/building-autoencoders-in-keras.html).

I will upload it very soon, but sorry again for the inconvenience...

**Comment by skyhoudou** [comment]
Hi, i am also very interested in this paper. And i meet across a problem.
Do I need to  normalize the stock_net_train before training the
auto-encoder?

**Comment by tcloaa** [comment]
Good question.

The paper does not mention about the data preprocessing, even the type of data (weekly return of what?: close price? percentage of change? net change?)

So I chose percentage of change and did not do any normalization. What do you think?

**Comment by skyhoudou** [comment]
exactly, i tried to train the percentage net change, but the result is not
good. Especially when I try to track the ibb index, the performance of the
predicted ibb index of selected stocks s25, s45 and s65 are bad. I am
trying to figure out if the parameters of neural networks are not suitable.
By the way, where did you find this data set? i want to collect more data
to see if the performance would be improved by enlarge the size of data
set.

**Comment by tcloaa** [comment]
@skyhoudou Excel + Bloomberg add-in, using VBA Macro.
